Week1/homework/index.html

```diff
@@ -2,15 +2,18 @@
 <html>
```

Comment:
Would love to see this somewhere, any chance you hosted it and I can view it with a link?

Reply:
https://outintotheblue.github.io/portfolio/js-2-1/homework/index.html
My small gaming website haha :)

Reply:
Nice, looks good! 🎮